_maps/RandomRuins/LavaRuins/lavaland_surface_ash_walker1.dmm

```diff
@@ -1168,9 +1168,9 @@
 /turf/open/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
 "dg" = (
-/obj/structure/bonfire/dense,
 /obj/structure/stone_tile/center,
 /obj/effect/mapping_helpers/no_lava,
+/obj/structure/bonfire/dense/askwalker,
```

Comment:
If this was done manually, you should move the structure/bonfire back to the top since the map generator does it in order and if this ever gets changed then that will be added to the next PR that changes it

Reply:
StrongDMM did it automatically, I didn't touch the inside of the file itself